### ISSUE TITLE
deletes too many images

### ISSUE THREAD

**Issue by kuntergunt** [therainisme/vscode-clear-markdown-unused-images#1]
I gave this extension a try on my Azure DevOps wiki. It found the images and moved them to the unused.images folder. But it moved more than 431 images out of 478. That seemed to be way too many for me. I checked and found a lot of pages with missing images. So somehow the algorithm does not detect every usage of an image. At least not for the Azure DevOps type of Markdown pages.

**Comment by therainisme** [comment]
Can you provide more information about the markdown of the removed images? For example, how to reference the image in markdown, and the location of the image.

**Comment by kuntergunt** [comment]
Here is an example, one of the pages that lost it's image:
The link in the source code looks like this: ![cert_exp.png](/.attachments/cert_exp-49141bcf-f2db-404c-b2a8-4fc7ede8b106.png)

The directory hierarchy for those two files in VSCode looks like this:
.attachments\cert_exp-49141bcf-f2db-404c-b2a8-4fc7ede8b106.png
Customer-Documentation\SAP-AddOn\Programs\Certificate-Expiration-Alert.md

**Comment by therainisme** [comment]
What kind of image link is in your `Customer-Documentation\SAP-AddOn\Programs\Certificate-Expiration-Alert.md` file?

```markdown
![image](.attachment\cert_exp-49141bcf-f2db-404c-b2a8-4fc7ede8b106.png)
```

```markdown
![image](../../../attachments/cert_exp-49141bcf-f2db-404c-b2a8-4fc7ede8b106.png)
```

The former seems to be a link format specific to the Azure DevOps wiki. I can't reproduce your bug in vscode.

**Comment by kuntergunt** [comment]
Sorry, seems like this part of the above message got modified. I try to make it readable, this is the part within the parenthesis:
/.attachments/cert_exp-49141bcf-f2db-404c-b2a8-4fc7ede8b106.png
So the path starts from root, but it's the root of the project, not of the file system. Seems weird, but all the tools I've used (including VSCode) seem to be able to understand this correctly,

**Comment by therainisme** [comment]
Thank you for reporting this issue! It has been fixed in #2.
